packages/@adobe/spectrum-css-temp/components/checkbox/index.css

```diff
@@ -30,7 +30,8 @@ governing permissions and limitations under the License.
   position: relative;
 
   min-block-size: var(--spectrum-checkbox-height);
-  max-inline-size: 100%;
+  /* Accommodates for right padding in case width: 100% is applied to the Checkbox directly. */
+  max-inline-size: calc(100% - var(--spectrum-checkbox-cursor-hit-x) * 2);
```

Comment:
???????????? Still digging to find out why this helps. At first I thought it was that the hit area was exceeding the container width and thus causing overflow to kick in but it doesn't truly seem to be the case. 

Fix works tho

Reply:
wha? but that element is absolutely positioned, it shouldn't affect the size at all
I do believe you that this works
did you try small sizes though? it may still fall apart there or the box size may looks funny for very wide cases (not that it would be bad in the wide case)

Reply:
yep, its not the hit area, it is actually just the checkbox's `margin-inline-end` padding that causes the checkbox to exceed the container width when `width: 100%` is applied directly to the Checkbox. I tried doing `box-sizing: border-box;` as was suggested in some posts I read but no dice. I figured the above change was preferable to removing the `margin-inline-end` padding since I felt people were more likely to put things next to checkboxes and since the above change just shortens the hit box a little bit

At small sizes the shortened checkbox is more noticeable for sure, but not horribly so IMO, especially at those widths you are dealing with single letter labels (which are still clickable actually):
![image](https://user-images.githubusercontent.com/9661127/76257174-6d3d0080-620e-11ea-9fb1-77ea9346cda9.png)
Uncaptured in the screenshot is my mouse hovering over the right edge of the "C" and it still changing into the hand(?) cursor that indicates a button/clickable element of some sort

Reply:
I would prefer having the `box-sizing: border-box` approach but can't for the life of me figure out why it isn't working